tests/load/test_s3.py

```diff
@@ -111,6 +112,40 @@ def test_s3_read_json_simple(benchmark_time):
     assert timer.elapsed_time < benchmark_time
 
 
+@pytest.mark.parametrize("benchmark_time", [15])
+def test_s3_write_csv(path: str, benchmark_time: int):
+    pandas_refs = ray.data.range_table(100_000).to_pandas_refs()
+    dataset = ray.data.from_pandas_refs(pandas_refs)
+
+    frame = dataset.to_modin()
+    frame["foo"] = frame.value * 2
+    frame["bar"] = frame.value % 2
```

Comment:
Nitpick - Since this code is already used twice, should we make it a fixture like it's for `df_s` and `df_xl` above?